.github/workflows/release-prepare.yml

```diff
@@ -259,11 +259,11 @@ jobs:
 
           cd "${HOLOCHAIN_REPO}"
 
-          git tag --sort=-taggerdate | grep holochain-
+          git tag --sort=-taggerdate --merged $(git branch --show-current) | grep holochain-
 
-          export LATEST_HOLOCHAIN_TAG=$(git tag --sort=-taggerdate | grep holochain- | head -n1)
-          export LATEST_HOLOCHAIN_VERSION=${LATEST_HOLOCHAIN_TAG/holochain-/}
           export RELEASE_BRANCH=$(git branch --show-current)
+          export LATEST_HOLOCHAIN_TAG=$(git tag --sort=-taggerdate --merged $RELEASE_BRANCH | grep holochain- | head -n1)
+          export LATEST_HOLOCHAIN_VERSION=${LATEST_HOLOCHAIN_TAG/holochain-/}
```

Comment:
technically it's not the absolutely latest version anymore using the branch filter. maybe we care enough about that to rename the variable?

Reply:
Hmm yes I see your point. It's the latest from the point of view of this release track but not in global context. I wonder how much value we could add by renaming this variable though because that's the way I understood this variable in the first place. I can't think what 1 or 2 words we could add

I'm happy to rename this if you have a better idea for a name?

I could also add a comment to the variable with a hint to the reader that the tag is scoped to the current release branch?